src/index.ts

```diff
@@ -30,10 +31,11 @@ export function cortina(
     throw new Error('Block api url must be specified.')
   }
   if (!redisConfig || !redisClient) {
-    return fetchAllBlocks(blocksConfig, blockApiUrl, language, headers)
+    return fetchAllBlocks(blocksConfig, blockApiUrl, language, headers, styleVersion)
   }
 
-  const { redisKey, redisExpire } = redisItemSettings
+  const { redisKey: redisKeyBase, redisExpire } = redisItemSettings
+  const redisKey = redisKeyBase + (styleVersion == 10 ? 'style10_' : 'style9_')
```

Comment:
~~I’m assuming the type coercion is by design. Is it to clever though? Easily missed by a tired developer on a Friday afternoon?~~ Wait, type is `number`. Shouldn’t this be `styleVersion == 10`?